src/resources/styles/settings.scss

```
@@ -6,8 +6,8 @@
 @import '~breakpoint-sass/stylesheets/breakpoint';
 
 // FONTS
-$font-family-sans-serif: Roboto, sans-serif;
-$font-family-divider: RobotoSlab, serif;
+$font-family-sans-serif: 'Inter Variable', Arial, sans-serif;
```

Stylelint (17.13.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)